test/traffic-2d.cpp

```diff
@@ -1004,10 +995,10 @@ class fltkview : public Fl_Gl_Window
 
             if(drawing)
             {
-                const vec2f low(make_v2(std::min(first_point[0], second_point[0]),
-                                        std::min(first_point[1], second_point[1])));
-                const vec2f high(make_v2(std::max(first_point[0], second_point[0]),
-                                         std::max(first_point[1], second_point[1])));
+                const vec2f low(std::min(first_point[0], second_point[0]),
```

Comment:
Does this compile? I remember having to work around this for the modern version of the code.

Reply:
It works on my machine. Yeah, but I will change it to `make_v2` in case it breaks the compilation.